internal/cli/importcmd/config.go

```diff
@@ -0,0 +1,29 @@
+package importcmd
+
+import (
+	"encoding/json"
+	"io/ioutil"
+)
+
+type Config struct {
+	Auth0Domain                 string                 `json:"AUTH0_DOMAIN"`
+	Auth0KeywordReplaceMappings map[string]interface{} `json:"AUTH0_KEYWORD_REPLACE_MAPPINGS"`
+	Auth0AllowDelete            string                 `json:"AUTH0_ALLOW_DELETE"`
```

Comment:
```suggestion
	Auth0AllowDelete            bool                   `json:"AUTH0_ALLOW_DELETE"`
```
My bad 🙈 , this should be a bool.